### DISCUSSION TITLE
Documentation About Ev Controllers and OpenCn

### DISCUSSION THREAD

**Discussion by akram09** [vrra/FGAN-Build-a-thon-2022#28] [Q&A (General)]
Hello, 
I was reading about the required steps for working on the build-a-thon, and didn't understand the concept of Evcontroller and openCN concepts, is there any documentation that explains these concepts?

**Comment by jhebus** [comment]
Hi there,

The evolution controller is responsible for driving the generation and modification of the design (specification) of a controller. How this is achieved (e.g. which algorithms, which parts of the controller to change, how often to change, which information is used to drive the approach, etc.) is specific to the implementation. This (design) task is usually performed by humans at the moment.

The openCN is meant to be an open location to store the different "bits" that would be required by an autonomous network. Examples may include controller specifications, generated controllers, use case definitions, experimental specifications, etc. 

The architecture document should help to answer these questions, however, please note that it is still in progress. https://extranet.itu.int/sites/itu-t/focusgroups/an/_layouts/15/WopiFrame.aspx?sourcedoc=%7BCABC48C0-0143-4677-8012-6A97797484D3%7D&file=FGAN-I-263-R2.docx&action=default&CT=1656930317710&OR=DocLibClassicUI
